atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/progress/SweepProgress.java

```diff
@@ -17,15 +17,21 @@
 
 import org.immutables.value.Value;
 
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
 import com.palantir.atlasdb.keyvalue.api.TableReference;
 
+@JsonSerialize(as = ImmutableSweepProgress.class)
+@JsonDeserialize(as = ImmutableSweepProgress.class)
 @Value.Immutable
 public interface SweepProgress {
 
     TableReference tableRef();
 
     byte[] startRow();
 
+    byte[] startColumn();
```

Comment:
We should probably add a comment that this is not currently used, but is present to avoid a future break as it's likely sweep will start using this information soon.